src/Sylius/Bundle/ApiBundle/Swagger/PathHiderDocumentationNormalizer.php

```diff
@@ -35,7 +35,7 @@ public function supportsNormalization($data, $format = null): bool
     public function normalize($object, $format = null, array $context = [])
     {
         $docs = $this->decoratedNormalizer->normalize($object, $format, $context);
-        $paths = $docs['paths']->getArrayCopy();
+        $paths = (array) $docs['paths'];
```

Comment:
FYI this is a potential bugfix. If the decoration priority is set to different number, the $docs sometimes can be a normal array (not ArrayObject), in this case getArrayCopy() throws error, but (array) does the same and does not make any issue.